<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Rick Gilmore" />


<title>PSY 511</title>

<script src="511-methods_files/header-attrs-2.9/header-attrs.js"></script>
<script src="511-methods_files/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="511-methods_files/bootstrap-3.3.5/css/lumen.min.css" rel="stylesheet" />
<script src="511-methods_files/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="511-methods_files/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="511-methods_files/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="511-methods_files/navigation-1.1/tabsets.js"></script>
<script src="511-methods_files/navigation-1.1/codefolding.js"></script>
<link href="511-methods_files/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="511-methods_files/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>




<style type="text/css">
/* for pandoc --citeproc since 2.11 */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="css/outline.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>



<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>




</head>

<body>


<div class="container-fluid main-container">




<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">PSY 511</h1>
<h3 class="subtitle">Methods</h3>
<h4 class="author">Rick Gilmore</h4>
<h4 class="date">2021-09-02 13:09:08</h4>

</div>

<div id="TOC">
<ul>
<li><a href="#neuroscience-methods">Neuroscience methods</a>
<ul>
<li><a href="#evaluating-methods">Evaluating methods</a>
<ul>
<li><a href="#what-are-we-measuring">What are we measuring?</a></li>
<li><a href="#what-is-the-question">What is the question?</a></li>
</ul></li>
<li><a href="#evaluating-methods-1">Evaluating methods</a>
<ul>
<li><a href="#strengths-weaknesses">Strengths &amp; Weaknesses</a></li>
</ul></li>
<li><a href="#spatial-resolution">Spatial resolution</a></li>
<li><a href="#and-temporal-resolution">…and temporal resolution</a></li>
<li><a href="#types-of-methods">Types of methods</a></li>
</ul></li>
<li><a href="#structural-methods">Structural methods</a>
<ul>
<li><a href="#mapping-microstructure">Mapping microstructure</a>
<ul>
<li><a href="#golgi-stain">Golgi stain</a>
<ul>
<li><a href="#camillo-golgi"><span>Camillo Golgi</span></a></li>
</ul></li>
<li><a href="#nissl-stain">Nissl stain</a>
<ul>
<li><a href="#franz-nissl"><span>Franz Nissl</span></a></li>
</ul></li>
<li><a href="#brainbow"><span>Brainbow</span></a></li>
<li><a href="#clarity"><span>Clarity</span></a></li>
<li><a href="#cubic"><span>CUBIC</span></a></li>
<li><a href="#example">Example</a></li>
<li><a href="#evaluating-microcellular-techniques">Evaluating micro/cellular techniques</a></li>
<li><a href="#example-1">Example</a></li>
</ul></li>
<li><a href="#mapping-macro-structures">Mapping macro-structures</a>
<ul>
<li><a href="#computed-axial-tomography-cat-ct">Computed axial tomography (CAT), CT</a>
<ul>
<li><a href="#tomography">Tomography</a></li>
</ul></li>
<li><a href="#magnetic-resonance-imaging-mri">Magnetic Resonance Imaging (MRI)</a>
<ul>
<li><a href="#what-it-measureshow-it-works">What it measures/how it works</a></li>
<li><a href="#structural-mri">Structural MRI</a></li>
<li><a href="#example-2">Example</a></li>
<li><a href="#mr-spectroscopy-specific-metabolites">MR Spectroscopy (specific metabolites)</a></li>
<li><a href="#voxel-based-morphometry-vbm">Voxel-based morphometry (VBM)</a></li>
</ul></li>
</ul></li>
<li><a href="#mapping-the-wiring-diagram-connectome">Mapping the wiring diagram (“connectome”)</a>
<ul>
<li><a href="#retrograde-output---input-vs.-anterograde-input---output-tracers">Retrograde (output -&gt; input) vs. anterograde (input -&gt; output) tracers</a></li>
<li><a href="#diffusion-tensor-imaging-dti">Diffusion Tensor Imaging (DTI)</a></li>
<li><a href="#visualizing-the-connectome">Visualizing the connectome</a></li>
</ul></li>
</ul></li>
<li><a href="#functional-methods">Functional methods</a>
<ul>
<li><a href="#recording-from-the-brain">Recording from the brain</a>
<ul>
<li><a href="#singlemulti-unit-recording">Single/multi-unit Recording</a></li>
<li><a href="#electrocorticography-ecog"><span>Electrocorticography (ECoG)</span></a></li>
<li><a href="#positron-emission-tomography-pet"><span>Positron Emission Tomography (PET)</span></a>
<ul>
<li><a href="#comparing-pet-and-fmri">Comparing PET and fMRI</a></li>
</ul></li>
</ul></li>
<li><a href="#functional-magnetic-resonance-imaging-fmri">Functional Magnetic Resonance Imaging (fMRI)</a>
<ul>
<li><a href="#evaluating-fmri">Evaluating fMRI</a></li>
<li><a href="#hemodynamic-response-function-hrf">Hemodynamic Response Function (HRF)</a></li>
<li><a href="#higher-field-strengths-3-tesla-vs.-7-tesla">Higher field strengths (3 Tesla vs. 7 Tesla)</a></li>
<li><a href="#but-fmri-underpowered">but fMRI underpowered</a></li>
<li><a href="#functional-near-infrared-spectroscopy-fnirs">Functional Near-infrared Spectroscopy (fNIRS)</a></li>
<li><a href="#electroencephalography-eeg">Electroencephalography (EEG)</a>
<ul>
<li><a href="#what-does-eeg-measure">What does EEG measure?</a></li>
<li><a href="#collecting-eeg">Collecting EEG</a></li>
<li><a href="#evaluating-eeg">Evaluating EEG</a></li>
<li><a href="#event-related-potentials-erps"><span>Event-related potentials (ERPs)</span></a></li>
<li><a href="#brain-computer-interface-bci"><span>Brain Computer Interface (BCI)</span></a></li>
</ul></li>
<li><a href="#magneto-encephalography-meg">Magneto-encephalography (MEG)</a></li>
<li><a href="#how-do-eegmeg-and-fmri-relate">How do EEG/MEG and fMRI relate?</a></li>
</ul></li>
<li><a href="#manipulating-the-brain">Manipulating the brain</a>
<ul>
<li><a href="#interfering-with-the-brain">Interfering with the brain</a>
<ul>
<li><a href="#phineas-gage">Phineas Gage</a></li>
<li><a href="#evaluating-neuropsychological-methods">Evaluating neuropsychological methods</a></li>
</ul></li>
</ul></li>
<li><a href="#stimulating-the-brain">Stimulating the brain</a>
<ul>
<li><a href="#deep-brain-stimulation-as-therapy">Deep brain stimulation as therapy</a></li>
<li><a href="#optogenetics"><span>Optogenetics</span></a></li>
</ul></li>
<li><a href="#simulating-the-brain"><em>Sim</em>ulating the brain</a>
<ul>
<li><a href="#blue-brain-project">Blue Brain project</a></li>
</ul></li>
<li><a href="#references">References</a></li>
</ul></li>
</ul>
</div>

<div id="neuroscience-methods" class="section level1">
<h1>Neuroscience methods</h1>
<div id="evaluating-methods" class="section level2">
<h2>Evaluating methods</h2>
<div id="what-are-we-measuring" class="section level3">
<h3>What are we measuring?</h3>
<ul>
<li>Structure</li>
<li>Activity
<ul>
<li>Why not <em>function</em>?</li>
</ul></li>
</ul>
</div>
<div id="what-is-the-question" class="section level3">
<h3>What is the question?</h3>
<ul>
<li>Structure X -&gt; Structure Y</li>
<li>Structure X -&gt; Function Y</li>
</ul>
</div>
</div>
<div id="evaluating-methods-1" class="section level2">
<h2>Evaluating methods</h2>
<div id="strengths-weaknesses" class="section level3">
<h3>Strengths &amp; Weaknesses</h3>
<ul>
<li>Cost</li>
<li>Invasiveness</li>
<li>Spatial/temporal resolution</li>
</ul>
</div>
</div>
<div id="spatial-resolution" class="section level2 flexbox vcenter">
<h2 class="flexbox vcenter">Spatial resolution</h2>
<div class="figure" style="text-align: center">
<img src="img/churchland-levels-of-analysis.gif" alt="http://ai.ato.ms/MITECS/Images/churchland.figure1.gif" width="600" />
<p class="caption">
<a href="http://ai.ato.ms/MITECS/Images/churchland.figure1.gif" class="uri">http://ai.ato.ms/MITECS/Images/churchland.figure1.gif</a>
</p>
</div>
</div>
<div id="and-temporal-resolution" class="section level2 flexbox vcenter">
<h2 class="flexbox vcenter">…and temporal resolution</h2>
<div class="centered">
<p><a href="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.3839/MediaObjects/41593_2014_Article_BFnn3839_Fig1_HTML.jpg?as=webp"> <img src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.3839/MediaObjects/41593_2014_Article_BFnn3839_Fig1_HTML.jpg?as=webp" width="600px"> </a></p>
<p><small> <a href="http://doi.org/10.1038/nn.3839"><span class="citation">(Sejnowski, Churchland, &amp; Movshon, 2014)</span></a> </small></p>
</div>
</div>
<div id="types-of-methods" class="section level2">
<h2>Types of methods</h2>
<ul>
<li>Structural
<ul>
<li>Anatomy</li>
<li>Connectivity/connectome</li>
</ul></li>
<li>Functional
<ul>
<li>What does it do?</li>
<li>Physiology/Activity</li>
</ul></li>
</ul>
</div>
</div>
<div id="structural-methods" class="section level1">
<h1>Structural methods</h1>
<div id="mapping-microstructure" class="section level2">
<h2>Mapping microstructure</h2>
<ul>
<li>Cell/axon stains</li>
<li>Cellular distribution, concentration, microanatomy</li>
</ul>
<div id="golgi-stain" class="section level3">
<h3>Golgi stain</h3>
<ul>
<li>whole cells, but small %</li>
</ul>
<div class="centered">
<p><img src="img/golgi-stain.jpg" width="612" height="500px" style="display: block; margin: auto;" /></p>
<p><a href="http://connectomethebook.com/wp-content/uploads/2011/11/Brainforest17_1119.jpg" class="uri">http://connectomethebook.com/wp-content/uploads/2011/11/Brainforest17_1119.jpg</a></p>
</div>
<hr />
<div class="centered">
<div class="figure" style="text-align: center">
<img src="http://wam.umn.edu/wp-content/uploads/2016/12/WAM_Cajal_m1673.jpg" alt="http://wam.umn.edu/wp-content/uploads/2016/12/WAM_Cajal_m1673.jpg" width="650px" />
<p class="caption">
<a href="http://wam.umn.edu/wp-content/uploads/2016/12/WAM_Cajal_m1673.jpg" class="uri">http://wam.umn.edu/wp-content/uploads/2016/12/WAM_Cajal_m1673.jpg</a>
</p>
</div>
<div id="camillo-golgi" class="section level4">
<h4><a href="https://en.wikipedia.org/wiki/Camillo_Golgi">Camillo Golgi</a></h4>
<p><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/5f/Camillo_Golgi.jpg/440px-Camillo_Golgi.jpg" height="550px" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="nissl-stain" class="section level3">
<h3>Nissl stain</h3>
<ul>
<li>Only cell bodies</li>
<li>Cell density ~ color intensity</li>
</ul>
<div class="centered">
<p><img src="https://i.pinimg.com/originals/24/ba/58/24ba5870a0b3ac2ce8e3620853e12c8b.jpg" height="500px" style="display: block; margin: auto;" /></p>
<div id="franz-nissl" class="section level4">
<h4><a href="https://en.wikipedia.org/wiki/Franz_Nissl">Franz Nissl</a></h4>
<div class="figure" style="text-align: center">
<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/d/db/Franz_Nissl.jpg/440px-Franz_Nissl.jpg" alt="https://en.wikipedia.org/wiki/Franz_Nissl" height="550px" />
<p class="caption">
<a href="https://en.wikipedia.org/wiki/Franz_Nissl" class="uri">https://en.wikipedia.org/wiki/Franz_Nissl</a>
</p>
</div>
</div>
</div>
<div id="brainbow" class="section level3">
<h3><a href="http://cbs.fas.harvard.edu/science/connectome-project/brainbow">Brainbow</a></h3>
<div class="figure" style="text-align: center">
<img src="img/lichtman-nrn2391-f1.jpg" alt="[[@lichtman_technicolour_2008]](http://doi.org/10.1038/nrn2391)" width="700px" />
<p class="caption">
<a href="http://doi.org/10.1038/nrn2391"><span class="citation">(Lichtman, Livet, &amp; Sanes, 2008)</span></a>
</p>
</div>
<div class="figure" style="text-align: center">
<img src="img/lichtman-nrn2391-f2.jpg" alt="[[@lichtman_technicolour_2008]](http://doi.org/10.1038/nrn2391)" width="550px" />
<p class="caption">
<a href="http://doi.org/10.1038/nrn2391"><span class="citation">(Lichtman, Livet, &amp; Sanes, 2008)</span></a>
</p>
</div>
</div>
<div id="clarity" class="section level3">
<h3><a href="http://clarityresourcecenter.com/CLARITY.html">Clarity</a></h3>
<iframe width="560" height="315" src="https://www.youtube.com/embed/c-NMfp13Uug" frameborder="0" allowfullscreen>
</iframe>
<p><a href="https://www.youtube.com/embed/c-NMfp13Uug" class="uri">https://www.youtube.com/embed/c-NMfp13Uug</a></p>
</div>
<div id="cubic" class="section level3">
<h3><a href="https://en.wikipedia.org/wiki/CUBIC">CUBIC</a></h3>
<ul>
<li>CUBIC (“clear, unobstructed brain/body imaging cocktails and computational analysis”)</li>
<li><a href="http://dx.doi.org/10.1016/j.cell.2014.03.042"><span class="citation">(Susaki et al., 2014)</span></a></li>
</ul>
</div>
<div id="example" class="section level3">
<h3>Example</h3>
<div class="figure" style="text-align: center">
<img src="https://ars.els-cdn.com/content/image/1-s2.0-B9780444636393000219-f21-02-9780444636393.jpg" alt="[[@Ando2018-ze]](https://doi.org/10.1016/B978-0-444-63639-3.00021-9)"  />
<p class="caption">
<a href="https://doi.org/10.1016/B978-0-444-63639-3.00021-9"><span class="citation">(Ando, Laborde, Brion, &amp; Duyckaerts, 2018)</span></a>
</p>
</div>
<blockquote>
<p>Fig. 21.2. Immunostaining of CUBIC-clarified 500-μm-thick slices from human Alzheimer disease postmortem brain frontal cortex. Human Alzheimer disease frontal cortex tissue immunostained for Aß (6E10, red) and for tau (B19, green). Stack depth of 264 μm; step size = 1 μm. Stack photos were taken with a two-photon microscope equipped with a 20 × air objective. Scale bar, 100 μm.</p>
</blockquote>
</div>
<div id="evaluating-microcellular-techniques" class="section level3">
<h3>Evaluating micro/cellular techniques</h3>
<ul>
<li>Invasive (in humans post-mortem only)</li>
<li>High <em>spatial</em> resolution, but poor/coarse <em>temporal</em></li>
</ul>
</div>
<div id="example-1" class="section level3">
<h3>Example</h3>
<p>“SHANK2 mutations associated with autism spectrum disorder cause hyperconnectivity of human neurons” <a href="http://dx.doi.org/10.1038/s41593-019-0365-8"><span class="citation">(Zaslavsky et al., 2019)</span></a></p>
<div class="figure" style="text-align: center">
<img src="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41593-019-0365-8/MediaObjects/41593_2019_365_Fig1_HTML.png?as=webp" alt="[[@Zaslavsky2019-ln]](http://dx.doi.org/10.1038/s41593-019-0365-8)"  />
<p class="caption">
<a href="http://dx.doi.org/10.1038/s41593-019-0365-8"><span class="citation">(Zaslavsky et al., 2019)</span></a>
</p>
</div>
<blockquote>
<p>a, iPSCs generated from multiple control and affected individuals are differentiated into NPCs. NPCs are differentiated in separate wells for 4 weeks and then differentially fluorescently labeled control (CTRL) and mutant (MUT) cells are sparsely seeded onto a large unlabeled neuronal population (the lawn) and cocultured with astrocytes. b, Timeline of the experiment, starting with seeding of NPCs. Measurements of mutant cells are normalized to control cells in the same well. c, Sparse seeding allows simultaneous analyses of cell morphology and connectivity (total number of SYN1 puncta) of single neurons. Scale bars, 100 μm. d, To compare cell morphology, paired representative traces are shown of control and SHANK2 ASD or engineered SHANK2 KO neurons grown in the same well. e, To compare synaptic function, sEPSCs are recorded from neurons grown in the same well. Confocal images and traces shown in c and d are representative of iPSC-derived neurons imaged in experiments depicted in Fig. 2a–c. sEPSC traces shown in e are representative of patch-clamp recordings of iPSC-derived neurons described in Fig. 3.</p>
</blockquote>
</div>
</div>
<div id="mapping-macro-structures" class="section level2">
<h2>Mapping macro-structures</h2>
<div id="computed-axial-tomography-cat-ct" class="section level3">
<h3>Computed axial tomography (CAT), CT</h3>
<ul>
<li>X-ray based</li>
</ul>
<div class="centered">
<p><img src="img/tomography.png" width="400px" style="display: block; margin: auto;" /></p>
<p><a href="http://img.tfd.com/mk/T/X2604-T-22.png" class="uri">http://img.tfd.com/mk/T/X2604-T-22.png</a></p>
</div>
<div id="tomography" class="section level4 flexbox vcenter">
<h4 class="flexbox vcenter">Tomography</h4>
<div class="centered">
<p><img src="img/cat-scan-pineapple.jpg" width="500px" style="display: block; margin: auto;" /></p>
<p><a href="http://static.howstuffworks.com/gif/cat-scan-pineapple.jpg" class="uri">http://static.howstuffworks.com/gif/cat-scan-pineapple.jpg</a></p>
</div>
<hr />
<!-- https://medium.com/datadriveninvestor/detecting-brain-hemorrhage-in-computed-tomography-ct-imaging-d1276cb6bdb7 -->
<div class="centered">
<p><img src="https://miro.medium.com/max/1024/1*j09PgBpdJIHXhlyjcby0HA.jpeg" width="700px" style="display: block; margin: auto;" /></p>
<p><small> <a href="https://medium.com/datadriveninvestor/detecting-brain-hemorrhage-in-computed-tomography-ct-imaging-d1276cb6bdb7" class="uri">https://medium.com/datadriveninvestor/detecting-brain-hemorrhage-in-computed-tomography-ct-imaging-d1276cb6bdb7</a> </small></p>
</div>
</div>
</div>
<div id="magnetic-resonance-imaging-mri" class="section level3">
<h3>Magnetic Resonance Imaging (MRI)</h3>
<div id="what-it-measureshow-it-works" class="section level4">
<h4>What it measures/how it works</h4>
<ul>
<li>Magnetic resonance a property of some isotopes and complex molecules</li>
<li>Hydrogen (<span class="math inline">\(H\)</span>), common in water &amp; fat, is one</li>
<li>In magnetic field, <span class="math inline">\(H\)</span> atoms absorb and release radio frequency (RF) energy</li>
<li><span class="math inline">\(H\)</span> atoms align with strong magnetic field</li>
</ul>
<hr />
<ul>
<li>Applying RF pulse perturbs alignment</li>
<li>Rate/timing of realignment varies by tissue</li>
<li>Realignment gives off radio frequency (RF) signals</li>
<li>Strength of RF ~ density of <span class="math inline">\(H\)</span> (or other target)</li>
<li>K-space (frequency/phase) -&gt; anatomical space
<ul>
<li><a href="https://en.wikipedia.org/wiki/Fourier_transform">Fourier analysis and synthesis</a> important for psychological/neural science, but possibly unfamiliar to many.</li>
</ul></li>
</ul>
<hr />
<div class="centered">
<p><img src="img/mri-steps.jpg" width="450px" style="display: block; margin: auto;" /></p>
<p><a href="http://s.hswstatic.com/gif/mri-steps.jpg" class="uri">http://s.hswstatic.com/gif/mri-steps.jpg</a></p>
</div>
</div>
<div id="structural-mri" class="section level4">
<h4>Structural MRI</h4>
<ul>
<li>Tissue density/type differences</li>
<li><strong>Gray matter</strong> (nerve cells &amp; <strong>dendrites</strong>) vs. <strong>white matter</strong> (<strong>axon fibers</strong>)</li>
</ul>
<p><img src="https://previews.123rf.com/images/ultimagaina/ultimagaina1405/ultimagaina140500005/28078449-Sequence-of-vertical-sections-of-a-human-brain-MRI-scan-Stock-Photo.jpg" height="550px" style="display: block; margin: auto;" /></p>
</div>
<div id="example-2" class="section level4">
<h4>Example</h4>
<div class="figure" style="text-align: center">
<img src="https://els-jbs-prod-cdn.jbs.elsevierhealth.com/cms/attachment/a462a956-7f5d-4995-9989-647b2823c66f/gr1_lrg.jpg" alt="[[@Lee2021-ae]](http://dx.doi.org/10.1016/j.biopsych.2020.10.014)"  />
<p class="caption">
<a href="http://dx.doi.org/10.1016/j.biopsych.2020.10.014"><span class="citation">(Lee et al., 2021)</span></a>
</p>
</div>
<blockquote>
<p><em>Figure 1</em>. Longitudinal trajectories of total cerebral volume, gray matter volume, and white matter volume from early to middle childhood (A) in boys with autism spectrum disorder (ASD) and typically developing (TD) boys and (B) in boys with ASD and disproportionate megalencephaly (ASD-DM), boys with ASD with normative cerebral volume-to-height ratio (ASD-N), and TD boys.</p>
</blockquote>
</div>
<div id="mr-spectroscopy-specific-metabolites" class="section level4">
<h4>MR Spectroscopy (specific metabolites)</h4>
<p><img src="https://images.radiopaedia.org/images/556479/0dc08a48892084b4c3b717feb9dfa4_big_gallery.jpg" height="550px" style="display: block; margin: auto;" /></p>
<ul>
<li>Region sizes/volumes</li>
</ul>
</div>
<div id="voxel-based-morphometry-vbm" class="section level4">
<h4>Voxel-based morphometry (VBM)</h4>
<ul>
<li>MRI technique for measuring brain sizes/volumes</li>
</ul>
<div class="figure" style="text-align: center">
<img src="https://openi.nlm.nih.gov/imgs/512/130/2927029/PMC2927029_mp2009146f1.png" alt="[[@Pomarol-Clotet2010-tq]](https://dx.doi.org/10.1038/mp.2009.146)" height="400px" />
<p class="caption">
<a href="https://dx.doi.org/10.1038/mp.2009.146"><span class="citation">(Pomarol-Clotet et al., 2010)</span></a>
</p>
</div>
<blockquote>
<p>Top panel: (a) voxel-based morphometry (VBM) findings. Regions showing significant volume reduction thresholded at P=0.01 in the schizophrenic patients are shown in orange. Bottom panel: (b) functional magnetic resonance imaging (fMRI) findings. Regions are shown where there were significant differences between patients and controls during performance of the n-back task (2-back vs baseline comparison), thresholded at P=0.01. Blue indicates hypoactivation, that is, areas where controls activated significantly more than the patients. Orange indicates areas where the schizophrenic patients showed failure to deactivate in comparison to controls. The right side of the images represents the left side of the brain.</p>
</blockquote>
<ul>
<li>Volume differences in schizophrenics vs. controls</li>
<li>Colored portions are statistical maps placed on top of a base structural map.</li>
<li>Maps (a) provide information about the comparison in brain volumes between patients and controls in those areas, and in (b) functional imaging differences in an n-back task.</li>
</ul>
</div>
</div>
</div>
<div id="mapping-the-wiring-diagram-connectome" class="section level2">
<h2>Mapping the wiring diagram (“connectome”)</h2>
<p><img src="https://i.stack.imgur.com/2OT3B.png" width="700px" style="display: block; margin: auto;" /></p>
<hr />
<p><img src="img/jonas-kording-2017.jpg" width="800px" style="display: block; margin: auto;" /></p>
<div id="retrograde-output---input-vs.-anterograde-input---output-tracers" class="section level3">
<h3>Retrograde (output -&gt; input) vs. anterograde (input -&gt; output) tracers</h3>
<p><img src="img/retrograde-anterograde-tracers.png" width="400px" style="display: block; margin: auto;" /></p>
<p><a href="http://openi.nlm.nih.gov/imgs/512/348/3176268/3176268_1471-2105-12-351-2.png" class="uri">http://openi.nlm.nih.gov/imgs/512/348/3176268/3176268_1471-2105-12-351-2.png</a></p>
<hr />
<iframe width="560" height="315" src="https://www.youtube.com/embed/nvXuq9jRWKE" frameborder="0" allowfullscreen>
</iframe>
<p><a href="https://www.youtube.com/embed/nvXuq9jRWKE" class="uri">https://www.youtube.com/embed/nvXuq9jRWKE</a></p>
</div>
<div id="diffusion-tensor-imaging-dti" class="section level3">
<h3>Diffusion Tensor Imaging (DTI)</h3>
<ul>
<li>Structural MRI technique</li>
<li>Diffusion tensor: measurement of spatial pattern of <span class="math inline">\(H_2O\)</span> diffusion in small volume</li>
<li>Uniform (“isotropic”) vs. non-uniform (“anisotropic”)</li>
<li>Strong anisotropy suggests large # of axons with similar orientations (fiber tracts)</li>
</ul>
<hr />
<p><img src="http://1.bp.blogspot.com/-YO3h0cRqTEc/UIRKEC_0N8I/AAAAAAAABQs/dEPzCbcyuCo/s1600/FA_tensor.png" width="700px" style="display: block; margin: auto;" /></p>
<hr />
<div class="figure" style="text-align: center">
<img src="https://www.nap.edu/openbook/13373/xhtml/images/p26.jpg" alt="https://www.nap.edu/openbook/13373/xhtml/images/p26.jpg"  />
<p class="caption">
<a href="https://www.nap.edu/openbook/13373/xhtml/images/p26.jpg" class="uri">https://www.nap.edu/openbook/13373/xhtml/images/p26.jpg</a>
</p>
</div>
</div>
<div id="visualizing-the-connectome" class="section level3">
<h3>Visualizing the connectome</h3>
<div class="figure" style="text-align: center">
<img src="https://journals.plos.org/plosone/article/figure/image?size=large&download=&id=10.1371/journal.pone.0135247.g002" alt="[[@Bonilha2015-qx]](http://dx.doi.org/10.1371/journal.pone.0135247)" height="500px" />
<p class="caption">
<a href="http://dx.doi.org/10.1371/journal.pone.0135247"><span class="citation">(Bonilha et al., 2015)</span></a>
</p>
</div>
<blockquote>
<p>Fig 2. Link-wise ICCs. Each matrix entry represents the ICC observed for the white matter link between the gray matter ROI in the row and the gray matter ROI in the column. <a href="https://doi.org/10.1371/journal.pone.0135247.g002" class="uri">https://doi.org/10.1371/journal.pone.0135247.g002</a></p>
<p><a href="http://dx.doi.org/10.1371/journal.pone.0135247"><span class="citation">(Bonilha et al., 2015)</span></a></p>
</blockquote>
<div class="figure" style="text-align: center">
<img src="http://www.humanconnectome.org/storage/app/media/news/2015/09/CCA_mode.jpg" alt="http://www.humanconnectome.org" width="600px" />
<p class="caption">
<a href="http://www.humanconnectome.org" class="uri">http://www.humanconnectome.org</a>
</p>
</div>
</div>
</div>
</div>
<div id="functional-methods" class="section level1">
<h1>Functional methods</h1>
<ul>
<li>Recording from the brain</li>
<li>Interfering with the brain</li>
<li>Stimulating the brain</li>
<li>Simulating the brain</li>
</ul>
<div id="recording-from-the-brain" class="section level2">
<h2>Recording from the brain</h2>
<div id="singlemulti-unit-recording" class="section level3 flexbox vcenter">
<h3 class="flexbox vcenter">Single/multi-unit Recording</h3>
<ul>
<li>Microelectrodes + amplification</li>
<li>Small numbers of nerve cells</li>
</ul>
<div class="figure" style="text-align: center">
<img src="img/multi-unit-recording.jpg" alt="https://www.nature.com/nrn/journal/v5/n11/images/nrn1535-i1.jpg" width="600px" />
<p class="caption">
<a href="https://www.nature.com/nrn/journal/v5/n11/images/nrn1535-i1.jpg" class="uri">https://www.nature.com/nrn/journal/v5/n11/images/nrn1535-i1.jpg</a>
</p>
</div>
<ul>
<li>What does neuron X respond to?</li>
<li>How does firing frequency, timing vary with behavior?</li>
<li>Great temporal (ms), spatial resolution (um)</li>
<li>Invasive</li>
<li>Rarely suitable for humans, but…</li>
</ul>
</div>
<div id="electrocorticography-ecog" class="section level3">
<h3><a href="https://en.wikipedia.org/wiki/Electrocorticography">Electrocorticography (ECoG)</a></h3>
<p><img src="https://www.researchgate.net/profile/Milena_Korostenskaja/publication/281907202/figure/fig2/AS:613868334223375@1523369024308/Grid-electrodes-A-Craniotomy-performed-for-electrocorticography-ECoG-grid-electrode_Q640.jpg" width="800px" style="display: block; margin: auto;" /></p>
<blockquote>
<p>Grid electrodes: (A) Craniotomy performed for electrocorticography (ECoG) grid electrode placement in epilepsy surgery candidate at Comprehensive Epilepsy Program, Florida Hospital for Children, Orlando, Florida, United States. (B) ECoG electrode grids placed directly on the brain surface. They will be used during presurgical monitoring for localizing seizure onset zone. The same electrodes are stimulated during electrical cortical stimulation mapping for identification of eloquent cortex. The ECoG signal recorded from these grids is separated in a different stream and used for real-time functional mapping (RTFM). (C) 3D reconstruction of the brain with overlaid grid electrodes. This reconstruction is used for creating RTFM montage.</p>
</blockquote>
<hr />
<iframe width="560" height="315" src="https://www.youtube.com/embed/u50HPRe3rOY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
</iframe>
<p><a href="https://www.youtube.com/watch?v=u50HPRe3rOY" class="uri">https://www.youtube.com/watch?v=u50HPRe3rOY</a></p>
</div>
<div id="positron-emission-tomography-pet" class="section level3">
<h3><a href="https://en.wikipedia.org/wiki/Positron_emission_tomography">Positron Emission Tomography (PET)</a></h3>
<iframe width="560" height="315" src="https://www.youtube.com/embed/GHLBcCv4rqk" frameborder="0" allowfullscreen>
</iframe>
<p><a href="https://www.youtube.com/embed/GHLBcCv4rqk" class="uri">https://www.youtube.com/embed/GHLBcCv4rqk</a></p>
<hr />
<ul>
<li>Radioactive tracers (glucose, oxygen)</li>
<li>Positron decay activates paired detectors</li>
<li>Tomographic techniques reconstruct 3D geometry</li>
<li>Experimental condition - control</li>
<li>Average across individuals</li>
<li>Temporal (~ s) and spatial (mm-cm) resolution worse than fMRI</li>
<li>Radioactive exposures + mildly invasive</li>
<li>Dose &lt; airline crew exposure in 1 yr</li>
</ul>
<div id="comparing-pet-and-fmri" class="section level4">
<h4>Comparing PET and fMRI</h4>
<div class="figure" style="text-align: center">
<img src="https://ars.els-cdn.com/content/image/1-s2.0-S105381191830586X-gr2_lrg.jpg" alt="[[@Rischka2018-pk](http://dx.doi.org/10.1016/j.neuroimage.2018.06.079)]"  />
<p class="caption">
[<span class="citation">(Rischka et al., 2018)</span>(<a href="http://dx.doi.org/10.1016/j.neuroimage.2018.06.079" class="uri">http://dx.doi.org/10.1016/j.neuroimage.2018.06.079</a>)]
</p>
</div>
<blockquote>
<p>Fig. 2. Task-specific changes during finger tapping and visual stimulation obtained with fPET and fMRI across all subjects. Good agreement between CMRGlu and BOLD was observed for primary motor and visual cortices. However, in secondary areas (e.g., supplementary motor area, cerebellum, secondary visual areas) significant changes were only detected with fMRI but not with fPET (Table 2). Statistical maps were corrected for multiple comparisons at p&lt;0.05 FWE corrected voxel-level.</p>
</blockquote>
<blockquote>
<p>The brain’s energy budget can be non-invasively assessed with different imaging modalities such as functional MRI (fMRI) and PET (fPET), which are sensitive to oxygen and glucose demands, respectively. The introduction of hybrid PET/MRI systems further enables the simultaneous acquisition of these parameters…The absence of a correlation and the different activation pattern between fPET and fMRI suggest that glucose metabolism and oxygen demand capture complementary aspects of energy demands.</p>
<p><a href="http://dx.doi.org/10.1016/j.neuroimage.2018.06.079"><span class="citation">(Rischka et al., 2018)</span></a>]</p>
</blockquote>
</div>
</div>
</div>
<div id="functional-magnetic-resonance-imaging-fmri" class="section level2">
<h2>Functional Magnetic Resonance Imaging (fMRI)</h2>
<ul>
<li>Neural activity -&gt; local <span class="math inline">\(O_2\)</span> consumption increase</li>
<li><em>Blood Oxygen Level Dependent (BOLD)</em> response</li>
<li>Oxygenated vs. deoxygenated hemoglobin ≠ magnetic susceptibility</li>
<li>How do regional blood <span class="math inline">\(O_2\)</span> levels (&amp; flow &amp; volume) vary with behavior X?</li>
<li>MRI “signals” relate to the speed (1/T) of “relaxation” of the perturbed nuclei to their state of alignment with the main (<span class="math inline">\(B_0\)</span>) magnetic field.</li>
<li>Imaging protocols emphasize different time constants of this relaxation (<span class="math inline">\(T1\)</span>, <span class="math inline">\(T2\)</span>, <span class="math inline">\(T2^*\)</span>); <span class="math inline">\(T^{2^*}\)</span> for BOLD imaging</li>
</ul>
<div id="evaluating-fmri" class="section level3">
<h3>Evaluating fMRI</h3>
<ul>
<li>Non-invasive, but expensive</li>
<li>Moderate but improving (mm) spatial, temporal (~sec) resolution</li>
<li>Spatial limits due to
<ul>
<li>field strength (@ 3T <span class="math inline">\(~3mm^3\)</span> voxel)</li>
<li>Physiology of hemodynamic response</li>
</ul></li>
<li>Temporal limits due to
<ul>
<li>Hemodynamic Response Function (HRF): ~ 1s delay plus 3-6 s ramp-up</li>
<li>Speed of image acquisition</li>
</ul></li>
<li><em>Indirect</em> measure of neural activity</li>
</ul>
</div>
<div id="hemodynamic-response-function-hrf" class="section level3 smaller">
<h3 class="smaller">Hemodynamic Response Function (HRF)</h3>
<div class="figure" style="text-align: center">
<img src="img/hemodynamic-response-function.png" alt="https://openi.nlm.nih.gov/imgs/512/236/3109590/3109590_TONIJ-5-24_F1.png" width="800px" />
<p class="caption">
<a href="https://openi.nlm.nih.gov/imgs/512/236/3109590/3109590_TONIJ-5-24_F1.png" class="uri">https://openi.nlm.nih.gov/imgs/512/236/3109590/3109590_TONIJ-5-24_F1.png</a>
</p>
</div>
<hr />
<div class="centered">
<p>Generate “predicted” BOLD response to event; compare to actual</p>
<p><img src="https://mriquestions.com/uploads/3/4/5/7/34572113/convolution-of-3-events_orig.gif" width="800px" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="higher-field-strengths-3-tesla-vs.-7-tesla" class="section level3">
<h3>Higher field strengths (3 Tesla vs. 7 Tesla)</h3>
<div class="figure" style="text-align: center">
<img src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3629563/bin/gr3.jpg" alt="[[@Sladky2013-bu]](https://dx.doi.org/10.1016/j.ejrad.2011.09.025)" height="450px" />
<p class="caption">
<a href="https://dx.doi.org/10.1016/j.ejrad.2011.09.025"><span class="citation">(Sladky et al., 2013)</span></a>
</p>
</div>
</div>
<div id="but-fmri-underpowered" class="section level3">
<h3>but fMRI underpowered</h3>
<div class="figure" style="text-align: center">
<img src="https://journals.plos.org/plosbiology/article/file?id=10.1371/journal.pbio.2000797.g003&type=large" alt="[[@Szucs2017-fc]]((https://doi.org/10.1371/journal.pbio.2000797))" width="700px" />
<p class="caption">
<a href="(https://doi.org/10.1371/journal.pbio.2000797)"><span class="citation">(Szucs &amp; Ioannidis, 2017)</span></a>
</p>
</div>
<blockquote>
<p>Assuming a realistic range of prior probabilities for null hypotheses, false report probability is likely to exceed 50% for the whole literature.</p>
<p><a href="(https://doi.org/10.1371/journal.pbio.2000797)"><span class="citation">(Szucs &amp; Ioannidis, 2017)</span></a></p>
</blockquote>
<hr />
<ul>
<li>Solutions
<ul>
<li>Make data, materials (analysis code) more widely and openly available</li>
<li><a href="https://openneuro.org">OpenNeuro.org</a>, <a href="https://www.humanconnectomeproject.org/">Human Connectome Project</a>, <a href="https://databrary.org">Databrary.org</a>, etc.</li>
<li>Reuse shared data (e.g., <a href="https://abcdstudy.org/">Adolescent Brain &amp; Cognitive Development (ABCD) Study</a>)</li>
<li>Increase sample sizes, improve detection of small effects</li>
</ul></li>
</ul>
</div>
<div id="functional-near-infrared-spectroscopy-fnirs" class="section level3">
<h3>Functional Near-infrared Spectroscopy (fNIRS)</h3>
<ul>
<li>Near infrared light penetrates scalp and skull, refracted by brain tissue</li>
<li>Returned signal altered by blood <span class="math inline">\(O_2\)</span> levels</li>
<li>Time course (temporal resolution) ~ BOLD fMRI</li>
<li>Spatial resolution low</li>
<li>More suitable for pediatric populations (less susceptible to movement artefact)</li>
</ul>
<hr />
<div class="figure" style="text-align: center">
<img src="https://cibsr.stanford.edu/NIRS_Lab/_jcr_content/main/panel_builder/panel_0/image.img.full.high.png" alt="Source: https://cibsr.stanford.edu/NIRS_Lab.html" width="700px" />
<p class="caption">
Source: <a href="https://cibsr.stanford.edu/NIRS_Lab.html" class="uri">https://cibsr.stanford.edu/NIRS_Lab.html</a>
</p>
</div>
<hr />
<div class="figure" style="text-align: center">
<img src="https://images.squarespace-cdn.com/content/v1/54e7b27de4b0b080e1552803/1557982253763-E0WULBZB7PQBS124L7JG/ke17ZwdGBToddI8pDm48kNNnzL9xRi0AnU-1PeZqRnNZw-zPPgdn4jUwVcJE1ZvWQUxwkmyExglNqGp0IvTJZamWLI2zvYWH8K3-s_4yszcp2ryTI0HqTOaaUohrI8PIN3GI4mGL6orgGNgjoMp7j5NF-bk__8b8xi11mt0OJKgKMshLAGzx4R3EDFOm1kBS/fNIRS-EEG+closeup+side+-+large.jpg?format=500w" alt="Source: https://nirx.net" width="600px" />
<p class="caption">
Source: <a href="https://nirx.net" class="uri">https://nirx.net</a>
</p>
</div>
</div>
<div id="electroencephalography-eeg" class="section level3">
<h3>Electroencephalography (EEG)</h3>
<ul>
<li>How does it work?</li>
<li>Electrodes on scalp or brain surface</li>
</ul>
<div id="what-does-eeg-measure" class="section level4">
<h4>What does EEG measure?</h4>
<ul>
<li>Voltage <em>differences</em> between source and reference electrode</li>
<li>Combined activity of huge # of neurons</li>
<li>Current/voltage gradients between <em>apical</em> (near surface) dendrites and <em>basal</em> (deeper) dendrites and cell body/soma</li>
</ul>
<div class="figure" style="text-align: center">
<img src="https://neurofeedbackalliance.org/wp-content/uploads/2016/10/Dipole.jpg" alt="https://neurofeedbackalliance.org/wp-content/uploads/2016/10/Dipole.jpg&quot;" width="800px" />
<p class="caption">
<a href="https://neurofeedbackalliance.org/wp-content/uploads/2016/10/Dipole.jpg" class="uri">https://neurofeedbackalliance.org/wp-content/uploads/2016/10/Dipole.jpg</a>"
</p>
</div>
</div>
<div id="collecting-eeg" class="section level4 flexbox vcenter smaller">
<h4 class="flexbox vcenter smaller">Collecting EEG</h4>
<div class="figure" style="text-align: center">
<img src="img/baby-eeg.jpg" alt="https://sfari.org/images/images-2013-folder/images-sfn-2013/20131110sfneeg" width="600px" />
<p class="caption">
<a href="https://sfari.org/images/images-2013-folder/images-sfn-2013/20131110sfneeg" class="uri">https://sfari.org/images/images-2013-folder/images-sfn-2013/20131110sfneeg</a>
</p>
</div>
</div>
<div id="evaluating-eeg" class="section level4">
<h4>Evaluating EEG</h4>
<ul>
<li>High temporal, poor spatial resolution</li>
<li>Analyze activity in different ‘bands’ of frequencies (via <a href="https://en.wikipedia.org/wiki/Fourier_transform">Fourier</a> analysis)
<ul>
<li>LOW: deep sleep (delta or <span class="math inline">\(\delta\)</span> band)</li>
<li>MIDDLE: Quiet, alert state (alpha <span class="math inline">\(\alpha\)</span> band)</li>
<li>HIGHER: Sensorimotor activity reflecting observed actions? (mu or <span class="math inline">\(\mu\)</span> band), <a href="https://dx.doi.org/10.1098/rsos.160662"><span class="citation">(Hobson &amp; Bishop, 2017)</span></a></li>
<li>HIGHER STILL: “Binding” information across senses or plasticity? (gamma or <span class="math inline">\(\gamma\)</span> band), <a href="https://dx.doi.org/10.1371/journal.pone.0186008"><span class="citation">(Amo et al., 2017)</span></a></li>
</ul></li>
</ul>
<hr />
<div class="figure" style="text-align: center">
<img src="img/eeg-frequency.jpg" alt="https://www.peakmind.co.uk/images/frequency.jpg" width="650px" />
<p class="caption">
<a href="https://www.peakmind.co.uk/images/frequency.jpg" class="uri">https://www.peakmind.co.uk/images/frequency.jpg</a>
</p>
</div>
</div>
<div id="event-related-potentials-erps" class="section level4">
<h4><a href="https://en.wikipedia.org/wiki/Event-related_potential">Event-related potentials (ERPs)</a></h4>
<ul>
<li>EEGs time-locked to some event</li>
<li>…Averaged over many such events (trials)</li>
</ul>
<p><img src="https://2.bp.blogspot.com/_2ob-1_LsjJs/TAUjw9i_dYI/AAAAAAAAAQQ/9AfiHsnD-P8/s1600/ERP_technique.gif" width="700px" style="display: block; margin: auto;" /></p>
</div>
<div id="brain-computer-interface-bci" class="section level4 smaller">
<h4 class="smaller"><a href="https://computer.howstuffworks.com/brain-computer-interface.htm">Brain Computer Interface (BCI)</a></h4>
<ul>
<li>Based on EEG/ERPs</li>
</ul>
<p><img src="https://scx1.b-cdn.net/csz/news/800/2015/562df18a48c5c.png" height="500px" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="magneto-encephalography-meg" class="section level3">
<h3>Magneto-encephalography (MEG)</h3>
<ul>
<li>Like EEG, but measuring magnetic fields</li>
<li>Electrical and magnetic fields orthogonal</li>
<li>High temporal resolution</li>
<li>Magnetic fields propagate w/o distortion
<ul>
<li>But are orthogonal to electric field</li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/3/35/Onde_electromagnetique.svg/220px-Onde_electromagnetique.svg.png" alt="https://en.wikipedia.org/wiki/Electromagnetic_field"  />
<p class="caption">
<a href="https://en.wikipedia.org/wiki/Electromagnetic_field" class="uri">https://en.wikipedia.org/wiki/Electromagnetic_field</a>
</p>
</div>
<ul>
<li>Requires shielded chamber (to keep out strong magnetic fields)</li>
<li>++ cost vs. EEG</li>
</ul>
<div class="figure" style="text-align: center">
<img src="https://upload.wikimedia.org/wikipedia/commons/e/e6/NIMH_MEG.jpg" alt="https://upload.wikimedia.org/wikipedia/commons/e/e6/NIMH_MEG.jpg" height="500px" />
<p class="caption">
<a href="https://upload.wikimedia.org/wikipedia/commons/e/e6/NIMH_MEG.jpg" class="uri">https://upload.wikimedia.org/wikipedia/commons/e/e6/NIMH_MEG.jpg</a>
</p>
</div>
New device minimizes problems with motion
<div class="figure" style="text-align: center">
<img src="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41467-019-12486-x/MediaObjects/41467_2019_12486_Fig1_HTML.png" alt="[[@Hill2019-ik]](https://doi.org/10.1038/s41467-019-12486-x)"  />
<p class="caption">
<a href="https://doi.org/10.1038/s41467-019-12486-x"><span class="citation">(Hill et al., 2019)</span></a>
</p>
</div>
<blockquote>
<p>Figure 1. A paediatric MEG system: a Experimental setup for three participants age 2- (left), 5- (centre) and 24-years (right). OPMs, housed in a modified bike helmet, measured the MEG signal. b Time-frequency spectra from a single (synthesised gradiometer) channel. Changes in neural oscillations are shown; blue indicates a reduction in oscillatory amplitude relative to baseline; yellow indicates an increase. Note reduction in beta (13–30 Hz) and mu (8–13 Hz) amplitude. c The spatial signature of beta modulation during the period of tactile stimulation (0 s &lt; t &lt; 2 s) (blue overlay)</p>
</blockquote>
</div>
<div id="how-do-eegmeg-and-fmri-relate" class="section level3">
<h3>How do EEG/MEG and fMRI relate?</h3>
<div class="centered">
<p><img src="img/logothetis-2001.jpg" width="800px" style="display: block; margin: auto;" /></p>
<p><a href="https://doi.org/10.1038/35084005"><span class="citation">(Logothetis, Pauls, Augath, Trinath, &amp; Oeltermann, 2001)</span></a></p>
</div>
<ul>
<li>BOLD fMRI likely reflects <strong>presynaptic</strong> <em>input</em> to area</li>
<li>EEG/MEG likely reflects <strong>postsynaptic</strong> <em>response</em> to those inputs</li>
<li><a href="https://doi.org/10.1038/35084005"><span class="citation">(Logothetis, Pauls, Augath, Trinath, &amp; Oeltermann, 2001)</span></a> and <a href="https://doi.org/10.1146/annurev.physiol.66.082602.092845"><span class="citation">(Logothetis &amp; Wandell, 2004)</span></a></li>
</ul>
</div>
</div>
<div id="manipulating-the-brain" class="section level2">
<h2>Manipulating the brain</h2>
<ul>
<li>Interfering with it</li>
<li>Stimulating it</li>
</ul>
<div id="interfering-with-the-brain" class="section level3">
<h3>Interfering with the brain</h3>
<ul>
<li>Nature’s“experiments”</li>
<li>Stroke, head injury, tumor</li>
<li>Neuropsychology</li>
</ul>
<div id="phineas-gage" class="section level4">
<h4>Phineas Gage</h4>
<div class="figure" style="text-align: center">
<img src="http://www.doctorsimpossible.com/wp-content/uploads/2017/11/Phineas-Gage-with-skull-picture.png" alt="http://www.doctorsimpossible.com/the-curious-case-of-phineas-gage/" height="500px" />
<p class="caption">
<a href="http://www.doctorsimpossible.com/the-curious-case-of-phineas-gage/" class="uri">http://www.doctorsimpossible.com/the-curious-case-of-phineas-gage/</a>
</p>
</div>
<hr />
<p><img src="https://upload.wikimedia.org/wikipedia/en/9/98/The_Man_Who_Mistook_His_Wife_for_a_Hat_cover.jpg" height="550px" style="display: block; margin: auto;" /></p>
</div>
<div id="evaluating-neuropsychological-methods" class="section level4">
<h4>Evaluating neuropsychological methods</h4>
<ul>
<li>Logic: IF damage to area X impairs performance, THEN region critical for behavior Y</li>
<li><em>Double dissociation</em>: Damage to area Z leaves behavior Y intact</li>
<li>Weak spatial/temporal resolution</li>
</ul>
</div>
</div>
</div>
<div id="stimulating-the-brain" class="section level2">
<h2>Stimulating the brain</h2>
<ul>
<li>Electrical (<strong>Direct Current Stimulation - DCS</strong>)</li>
<li>Pharmacological</li>
<li>Magnetic (<strong>Transcranial magnetic stimulation - TMS</strong>)</li>
<li>Spatial/temporal resolution?</li>
<li>Assume stimulation mimics natural activity?</li>
</ul>
<div id="deep-brain-stimulation-as-therapy" class="section level4">
<h4>Deep brain stimulation as therapy</h4>
<ul>
<li>Depression</li>
<li>Epilepsy</li>
<li>Parkinson’s Disease</li>
</ul>
<div class="figure" style="text-align: center">
<img src="img/deep-brain-stimulation.jpg" alt="https://www.nimh.nih.gov/images/health-and-outreach/mental-health-topic-brain-stimulation-therapies/dbs_60715_3.jpg" width="700px" />
<p class="caption">
<a href="https://www.nimh.nih.gov/images/health-and-outreach/mental-health-topic-brain-stimulation-therapies/dbs_60715_3.jpg" class="uri">https://www.nimh.nih.gov/images/health-and-outreach/mental-health-topic-brain-stimulation-therapies/dbs_60715_3.jpg</a>
</p>
</div>
<hr />
<iframe width="560" height="315" src="https://www.youtube.com/embed/KDjWdtDyz5I" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen>
</iframe>
<p><a href="https://youtu.be/KDjWdtDyz5I" class="uri">https://youtu.be/KDjWdtDyz5I</a></p>
</div>
<div id="optogenetics" class="section level3">
<h3><a href="https://en.wikipedia.org/wiki/Optogenetics">Optogenetics</a></h3>
<iframe width="560" height="315" src="https://www.youtube.com/embed/I64X7vHSHOE" frameborder="0" allowfullscreen>
</iframe>
<p><a href="https://www.youtube.com/embed/I64X7vHSHOE" class="uri">https://www.youtube.com/embed/I64X7vHSHOE</a></p>
<ul>
<li>Gene splicing techniques insert light-sensitive molecules into neuronal membranes</li>
<li>Application of light at specific wavelengths alters neuronal function</li>
<li>Cell-type specific and temporally precise control</li>
<li>Mimics brain activity</li>
</ul>
<iframe width="560" height="315" src="https://www.youtube.com/embed/FlGbznBmx8M" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen>
</iframe>
<p><a href="https://youtu.be/FlGbznBmx8M" class="uri">https://youtu.be/FlGbznBmx8M</a></p>
</div>
</div>
<div id="simulating-the-brain" class="section level2">
<h2><em>Sim</em>ulating the brain</h2>
<ul>
<li>Computer/mathematical models of brain function</li>
<li>Example: neural networks</li>
<li>Cheap, noninvasive, can be stimulated or “lesioned”</li>
</ul>
<div id="blue-brain-project" class="section level3">
<h3>Blue Brain project</h3>
<div class="figure" style="text-align: center">
<img src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnrn1848/MediaObjects/41583_2006_Article_BFnrn1848_Fig4_HTML.jpg?as=webp" alt="[[@Markram2006-vu]](http://dx.doi.org/10.1038/nrn1848)" width="600px" />
<p class="caption">
<a href="http://dx.doi.org/10.1038/nrn1848"><span class="citation">(Markram, 2006)</span></a>
</p>
</div>
<hr />
<iframe width="560" height="315" src="https://www.youtube.com/embed/gn4nRCC9TwQ" frameborder="0" allowfullscreen>
</iframe>
<p><a href="https://www.youtube.com/embed/gn4nRCC9TwQ" class="uri">https://www.youtube.com/embed/gn4nRCC9TwQ</a></p>
</div>
</div>
<div id="references" class="section level2 unnumbered">
<h2 class="unnumbered">References</h2>
<div id="refs" class="references csl-bib-body hanging-indent" line-spacing="2">
<div id="ref-Amo2017-tz" class="csl-entry">
Amo, C., De Santiago, L., Zarza Luciáñez, D., León Alonso-Cortés, J. M., Alonso-Alonso, M., Barea, R., &amp; Boquete, L. (2017). Induced gamma band activity from <span>EEG</span> as a possible index of training-related brain plasticity in motor tasks. <em>PloS One</em>, <em>12</em>(10), e0186008. <a href="https://doi.org/10.1371/journal.pone.0186008">https://doi.org/10.1371/journal.pone.0186008</a>
</div>
<div id="ref-Ando2018-ze" class="csl-entry">
Ando, K., Laborde, Q., Brion, J.-P., &amp; Duyckaerts, C. (2018). Chapter 21 - <span>3D</span> imaging in the postmortem human brain with <span>CLARITY</span> and <span>CUBIC</span>. In I. Huitinga &amp; M. J. Webster (Eds.), <em>Handbook of clinical neurology</em> (Vol. 150, pp. 303–317). Elsevier. <a href="https://doi.org/10.1016/B978-0-444-63639-3.00021-9">https://doi.org/10.1016/B978-0-444-63639-3.00021-9</a>
</div>
<div id="ref-Bonilha2015-qx" class="csl-entry">
Bonilha, L., Gleichgerrcht, E., Fridriksson, J., Rorden, C., Breedlove, J. L., Nesland, T., … Focke, N. K. (2015). Reproducibility of the structural brain connectome derived from diffusion tensor imaging. <em>PloS One</em>, <em>10</em>(8), e0135247. <a href="https://doi.org/10.1371/journal.pone.0135247">https://doi.org/10.1371/journal.pone.0135247</a>
</div>
<div id="ref-Hill2019-ik" class="csl-entry">
Hill, R. M., Boto, E., Holmes, N., Hartley, C., Seedat, Z. A., Leggett, J., … Brookes, M. J. (2019). A tool for functional brain imaging with lifespan compliance. <em>Nature Communications</em>, <em>10</em>(1), 4785. <a href="https://doi.org/10.1038/s41467-019-12486-x">https://doi.org/10.1038/s41467-019-12486-x</a>
</div>
<div id="ref-Hobson2017-oj" class="csl-entry">
Hobson, H. M., &amp; Bishop, D. V. M. (2017). The interpretation of mu suppression as an index of mirror neuron activity: Past, present and future. <em>Royal Society Open Science</em>, <em>4</em>(3), 160662. <a href="https://doi.org/10.1098/rsos.160662">https://doi.org/10.1098/rsos.160662</a>
</div>
<div id="ref-Lee2021-ae" class="csl-entry">
Lee, J. K., Andrews, D. S., Ozonoff, S., Solomon, M., Rogers, S., Amaral, D. G., &amp; Nordahl, C. W. (2021). Longitudinal evaluation of cerebral growth across childhood in boys and girls with autism spectrum disorder. <em>Biological Psychiatry</em>, <em>90</em>(5), 286–294. <a href="https://doi.org/10.1016/j.biopsych.2020.10.014">https://doi.org/10.1016/j.biopsych.2020.10.014</a>
</div>
<div id="ref-lichtman_technicolour_2008" class="csl-entry">
Lichtman, J. W., Livet, J., &amp; Sanes, J. R. (2008). A technicolour approach to the connectome. <em>Nature Reviews Neuroscience</em>, <em>9</em>(6), 417–422. <a href="https://doi.org/10.1038/nrn2391">https://doi.org/10.1038/nrn2391</a>
</div>
<div id="ref-Logothetis2001-ul" class="csl-entry">
Logothetis, N. K., Pauls, J., Augath, M., Trinath, T., &amp; Oeltermann, A. (2001). Neurophysiological investigation of the basis of the <span class="nocase">fMRI</span> signal. <em>Nature</em>, <em>412</em>(6843), 150–157. <a href="https://doi.org/10.1038/35084005">https://doi.org/10.1038/35084005</a>
</div>
<div id="ref-Logothetis2004-mn" class="csl-entry">
Logothetis, N. K., &amp; Wandell, B. A. (2004). Interpreting the <span>BOLD</span> signal. <em>Annu. Rev. Physiol.</em>, <em>66</em>(1), 735–769. <a href="https://doi.org/10.1146/annurev.physiol.66.082602.092845">https://doi.org/10.1146/annurev.physiol.66.082602.092845</a>
</div>
<div id="ref-Markram2006-vu" class="csl-entry">
Markram, H. (2006). The blue brain project. <em>Nature Reviews. Neuroscience</em>, <em>7</em>(2), 153–160. <a href="https://doi.org/10.1038/nrn1848">https://doi.org/10.1038/nrn1848</a>
</div>
<div id="ref-Pomarol-Clotet2010-tq" class="csl-entry">
Pomarol-Clotet, E., Canales-Rodrı́guez, E. J., Salvador, R., Sarró, S., Gomar, J. J., Vila, F., … McKenna, P. J. (2010). Medial prefrontal cortex pathology in schizophrenia as revealed by convergent findings from multimodal imaging. <em>Mol. Psychiatry</em>, <em>15</em>(8), 823–830. <a href="https://doi.org/10.1038/mp.2009.146">https://doi.org/10.1038/mp.2009.146</a>
</div>
<div id="ref-Rischka2018-pk" class="csl-entry">
Rischka, L., Gryglewski, G., Pfaff, S., Vanicek, T., Hienert, M., Klöbl, M., … Hahn, A. (2018). Reduced task durations in functional <span>PET</span> imaging with [<span>18F]FDG</span> approaching that of functional <span>MRI</span>. <em>NeuroImage</em>, <em>181</em>, 323–330. <a href="https://doi.org/10.1016/j.neuroimage.2018.06.079">https://doi.org/10.1016/j.neuroimage.2018.06.079</a>
</div>
<div id="ref-Sejnowski2014-aa" class="csl-entry">
Sejnowski, T. J., Churchland, P. S., &amp; Movshon, J. A. (2014). Putting big data to good use in neuroscience. <em>Nat. Neurosci.</em>, <em>17</em>(11), 1440–1441. <a href="https://doi.org/10.1038/nn.3839">https://doi.org/10.1038/nn.3839</a>
</div>
<div id="ref-Sladky2013-bu" class="csl-entry">
Sladky, R., Baldinger, P., Kranz, G. S., Tröstl, J., Höflich, A., Lanzenberger, R., … Windischberger, C. (2013). High-resolution functional <span>MRI</span> of the human amygdala at 7 <span>T</span>. <em>Eur. J. Radiol.</em>, <em>82</em>(5), 728–733. <a href="https://doi.org/10.1016/j.ejrad.2011.09.025">https://doi.org/10.1016/j.ejrad.2011.09.025</a>
</div>
<div id="ref-Susaki2014-xj" class="csl-entry">
Susaki, E. A., Tainaka, K., Perrin, D., Kishino, F., Tawara, T., Watanabe, T. M., … Ueda, H. R. (2014). Whole-brain imaging with single-cell resolution using chemical cocktails and computational analysis. <em>Cell</em>, <em>157</em>(3), 726–739. <a href="https://doi.org/10.1016/j.cell.2014.03.042">https://doi.org/10.1016/j.cell.2014.03.042</a>
</div>
<div id="ref-Szucs2017-fc" class="csl-entry">
Szucs, D., &amp; Ioannidis, J. P. A. (2017). Empirical assessment of published effect sizes and power in the recent cognitive neuroscience and psychology literature. <em>PLoS Biol.</em>, <em>15</em>(3), e2000797. <a href="https://doi.org/10.1371/journal.pbio.2000797">https://doi.org/10.1371/journal.pbio.2000797</a>
</div>
<div id="ref-Zaslavsky2019-ln" class="csl-entry">
Zaslavsky, K., Zhang, W.-B., McCready, F. P., Rodrigues, D. C., Deneault, E., Loo, C., … Ellis, J. (2019). <span>SHANK2</span> mutations associated with autism spectrum disorder cause hyperconnectivity of human neurons. <em>Nature Neuroscience</em>, <em>22</em>(4), 556–564. <a href="https://doi.org/10.1038/s41593-019-0365-8">https://doi.org/10.1038/s41593-019-0365-8</a>
</div>
</div>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
